config/src/vespa/config/subscription/configsubscriptionset.cpp

```diff
@@ -76,6 +76,8 @@ ConfigSubscriptionSet::acquireSnapshot(uint64_t timeoutInMillis, bool ignoreChan
         timeLeft = timeoutInMillis - static_cast<uint64_t>(timer.MilliSecsToNow());
         if (!inSync && timeLeft > 0) {
             std::this_thread::sleep_for(10ms);
+        } else {
+            break;
         }
```

Comment:
I do not think this one is necessary since the next is the while() statement that will immediately break out anyway.
I just adds mental complexity.

Reply:
if inSync is false and timeLeft == 0 then the while() statement will not break out.

Reply:
Yes, I know, but this will only avoid an extra iteration when it is on the verge of timing out.